release_notes.md

```diff
@@ -1,9 +1,38 @@
 # What's New
- 
-## Internationalization
 
-Added support for following languages:
+## **Simplified Setup with the Configuration Wizard**
+Effortlessly set up your Threat Intel Management system with our new **Configuration Wizard**. This step-by-step guide ensures an intuitive and smooth setup experience by breaking tasks into manageable stages:
+- **Feed Source Selection**: Easily choose the feed sources most relevant to your needs.
+- **Integration Installation & Configuration**: Quickly install and configure integrations with just a few clicks.
+- **Granular Rule Selection**: Leverage powerful rules to manage feeds, even those with unstructured data.
```

Comment:
Points to be added;
1. MITRE connector getting installed and configured by default
2. SVT is also updated for the new fields
3. First class dashboard is got added as previous to a dashboard within the SVT

Reply:
Done